src/main/java/uk/gov/hmcts/ccd/endpoint/exceptions/DataProcessingException.java

```diff
@@ -0,0 +1,12 @@
+package uk.gov.hmcts.ccd.endpoint.exceptions;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(code = HttpStatus.UNPROCESSABLE_ENTITY)
+public class DataProcessingException extends ValidationException {
```

Comment:
why DataProcessing exception should be a subclass of ValidationException? in theory we could have processing errors that have nothing to do with validation